extension/src/Documentation.ts

```diff
@@ -678,4 +744,21 @@ function codeBlock(code: string): string {
     return result;
 }
 
+function table(innerHtml: string): string {
+    return `${tag('table', innerHtml, true)}\n`;
```

Comment:
Is this this extra newline intentional? (Since a newline is added in `tag()`)

Reply:
Yes, need an empty line after the table, according to MD inting rules..

Reply:
Yes ofc, me not thinking all the way :)